rust/Cargo.toml

```diff
@@ -12,3 +12,7 @@ resolver = "2"
 
 [patch.crates-io]
 onig = { git = "https://github.com/rust-onig/rust-onig", default-features = false }
+pact_models = { path = "./pact_models" }
+pact_matching = { path = "./pact_matching" }
+pact_mock_server = { git = 'https://github.com/pact-foundation/pact-core-mock-server.git', branch = "issue/134_missing_buffer_logs" }
```

Comment:
note these are reliant on patches in branches which pull in the updated pact-models code from this pr.

will need co-ordinating with release of pact models, then pact-plugin-driver, which is required by pact-core-mock-server, which is required by crates in this repo.